### PR TITLE
Add STARCounts data

### LINKED ISSUE #56: Add STARCounts data for TARGET cohorts

### PR DESCRIPTION
**Pull request by ayan-b**
Closes #56

### CODE REVIEW

File: xena_gdc_etl/resources/template.starcounts.meta.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{"cohort":"{{ xena_cohort }}",
+ "url":"{% if gdc_release %}{{ gdc_release }}, {% endif %}https://api.gdc.cancer.gov/data/",
+ "dataProducer":"Genomic Data Commons",
+ "label":"{{ Workflow_Type }}",
+ "colNormalization":true,
```
Comment (ayan-b) [inline xena_gdc_etl/resources/template.starcounts.meta.json]: Should this be removed?

File: xena_gdc_etl/resources/template.starcounts.meta.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+ "wrangling_procedure":
+ "Data from the same sample but from different vials/portions/analytes/aliquotes is averaged; data from different samples is combined into genomicMatrix; all data is then log2(x+1) transformed.",
+ "type":"genomicMatrix",
+ "wrangler":"Xena GDC ETL script",
+ "version":"{{ date }}",
+ "unit":"{% if unit %}log2({{ unit }}+1){% else %}log2(count+1){% endif %}"{% if notes %},
+ "notes":"{{ notes }}"{% endif %}
+}
```
[the remark this answers is not in the document]
Comment (ayan-b) [reply xena_gdc_etl/resources/template.starcounts.meta.json]: Using `rna template` now.

**Comment by yunhailuo** [comment]
Good to go. Can you make one matrix with 31331a7156cf8d7923e08a9f299a9ef7aafcabf7 and let's do a quick check?

**Comment by ayan-b** [comment]
Already made the matrix and json file and checked. BTW, this is how json looks:
```json
{"cohort":"GDC TARGET-ALL-P3",
 "url":"https://docs.gdc.cancer.gov/Data/Release_Notes/Data_Release_Notes/#data-release-170, https://api.gdc.cancer.gov/data/",
 "dataSubType":"gene expression RNAseq",
 ":probeMap":"gencode.v22.annotation.gene.probeMap",
 "dataProducer":"Genomic Data Commons",
 "label":"STAR - Counts",
 "colNormalization":true,
 "wrangling_procedure":
 "Data from the same sample but from different vials/portions/analytes/aliquotes is averaged; data from different samples is combined into genomicMatrix; all data is then log2(x+1) transformed.",
 "PLATFORM":"Illumina",
 "type":"genomicMatrix",
 "wrangler":"Xena GDC ETL script",
 "version":"07-02-2019",
 "unit":"log2(count+1)",
 "GDC_Description": "https://docs.gdc.cancer.gov/Data/Bioinformatics_Pipelines/Expression_mRNA_Pipeline/"
}
```

**Comment by ayan-b** [comment]
Loaded data into the hub.

**Pull request merged**